src/javascript/gameInstance.js

```diff
@@ -72,6 +72,7 @@ export default class GameInstance {
 
     this.trackNum = trackNum;
     this.trackKeyBind = this.getTrackKeyBind(trackNum);
+    this.setUserKeyBind(this.trackKeyBind);
```

Comment:
I mean if you can access `this.trackKeyBind`, maybe you don't want to pass it as parameter, and just use the instance member? I.e.

`this.setUserKeyBind()` but use `this.trackKeyBind` inside of the method.